rllib/utils/from_config.py

```diff
@@ -143,17 +143,26 @@ def from_config(cls, config=None, **kwargs):
             else:
                 return obj
 
-            # Test for absolute module.class specifier.
+            # Test for absolute module.class path specifier.
             if type_.find(".") != -1:
                 module_name, function_name = type_.rsplit(".", 1)
                 try:
                     module = importlib.import_module(module_name)
                     constructor = getattr(module, function_name)
-                except (ModuleNotFoundError, ImportError):
+                # Module not found.
+                except (ModuleNotFoundError, ImportError, AttributeError):
                     pass
+
             # If constructor still not found, try attaching cls' module,
             # then look for type_ in there.
             if constructor is None:
+                if isinstance(cls, str):
+                    # Module found, but doesn't have the specified
+                    # c'tor/function.
+                    raise ValueError(
+                        "String specifier ({}) in `from_config` must be a "
```

Comment:
nit, do users who consume this error message know what from_config is?
should we just leave it out, and also give an example at the end, like:

Path specifier ({}) must be a valid full modele.Class specifier (e.g. gym.envs.MyEnv)!

Reply:
Great point! The `from_config` in the message might confuse them. I'll change it.